core/src/main/java/google/registry/flows/domain/DomainPricingLogic.java

```diff
@@ -102,6 +107,7 @@ FeesAndCredits getCreatePrice(
             .build());
   }
 
+  // TODO (rachelguan): this will be replaced by the method getRenewPrice below in PR#1592
```

Comment:
Can you just have one `getRenewPrice()` method?  Seeing as how the existing one is unused, I don't see why you're retaining it.  Just delete it and replace it with the new one.